README.md

```diff
@@ -1,5 +1,7 @@
 ![GitHub](https://img.shields.io/github/license/decline-cookies/anvil-csharp-core)
-[![CircleCI](https://circleci.com/gh/decline-cookies/anvil-csharp-core/tree/main.svg?style=shield&circle-token=75e18a1354e8eb105cb7601a6f640b23e6975da4)](https://circleci.com/gh/decline-cookies/anvil-csharp-core/tree/main)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/decline-cookies/anvil-csharp-core/.NET%20Build?label=.NET%20Build)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/decline-cookies/anvil-csharp-core/MSBuild?label=MSBuild)
+
```

Comment:
I've never worked with shields, but hopefully this is right? Not sure how to test until the workflow is actually running on the main branch.

Reply:
Never mind, thought shields.io was the definitive source of project status shields, but Mike pointed me to the official GitHub workflow badges, which are now set up.